decoder_test.go

```diff
@@ -315,6 +319,42 @@ func TestDecodeColumn(t *testing.T) {
 			value: []spanner.NullJSON(nil),
 			want:  "NULL",
 		},
+
+		// PROTO
+		// This table tests only have null proto cases because of non-stability
+		// See also TestDecodeColumnGCV
+		{
+			desc:  "null proto",
+			value: (*protos.SingerInfo)(nil),
+			want:  "NULL",
+		},
```

Comment:
Maybe we can replace this with the following test to stop dependency on the imported proto?

```
		{
			desc: "null proto",
			value: spanner.GenericColumnValue{
				Type: &sppb.Type{
					Code: sppb.TypeCode_PROTO,
				},
				Value: structpb.NewNullValue(),
			},
			want: "NULL",
		},
```